src/ds/prob/top-k.md

```diff
@@ -0,0 +1,41 @@
+In this tutorial you will learn how to use the Top-K data structure in a bike shop use case.
+
+A Top-K maintains a list of *K* most frequently seen items.
+
+To get started, initialize a Top-K with a width (the number of counters kept in each array), depth (the number of arrays), and decay (the probability of reducing a counter in an occupied bucket).
```

Comment:
Please add a reference to an external source that explains these terms (or mention that such a source is given below)

Reply:
Please provide a reference.